tests/testthat/test-augment.R

```diff
@@ -26,6 +26,8 @@ test_that("augment fit_resamples", {
   expect_true(sum(names(aug_1) == ".resid") == 0)
   expect_s3_class_bare_tibble(aug_1)
 
+  expect_true(names(aug_1)[1] == ".pred_class")
+
   expect_identical(
     rle(grepl("^\\.", names(aug_1)))$values,
```

Comment:
Just out of curiosity, do we know what this is testing?

Reply:
if you are referring to line 32, it test that columns that start with `.` comes first https://github.com/tidymodels/tune/pull/799